crates/apollo-compiler/src/ast/serialize.rs

```diff
@@ -98,6 +98,19 @@ impl<'config, 'fmt, 'fmt2> State<'config, 'fmt, 'fmt2> {
         Ok(())
     }
 
+    /// Panics if newlines are disabled
+    fn require_new_line(&mut self) -> fmt::Result {
+        let prefix = self
+            .config
+            .ident_prefix
```

Comment:
Oh I was confused reading this, but it's meant to be `iNdent_prefix`

Reply:
Eh, it’s been wrong all this time :sweat_smile: